### ISSUE TITLE
Teams Adaptive Cards Not Displaying All Elements in Bot Framework

### ISSUE THREAD

**Issue by deeptishashidharaiah** [MicrosoftDocs/msteams-docs#11278]
### Steps to reproduce

![image](https://github.com/user-attachments/assets/07e41aa0-94e4-4c64-a7a9-e9e4d0b438b9)
We are experiencing an issue where adaptive cards in Microsoft Teams, created through the Bot Framework, are not displaying all elements. Specifically, we have a card with 15+ buttons that displays correctly on a webpage, but when viewed in the Teams app, it only shows up to 4 buttons.

### Expected behavior

All 15+ buttons should be displayed in the adaptive card within the Teams app, as they are on the webpage.

### Actual behavior

Only 4 buttons are displayed in the adaptive card within the Teams app, with the rest not visible.

**Comment by Meghana-MSFT** [comment]
Thank you for reporting this, we will check this and get back to you.

**Comment by Meghana-MSFT** [comment]
@deeptishashidharaiah - Please note that in Teams up to 6 button actions are supported. Remaining buttons will be shown as secondary actions. 

https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/design-effective-cards?tabs=design#actionset:~:text=Primary%20actions%3A-,As%20a%20best%20practice%2C%20include%201%2D3%20primary%20actions.%20You%20can%20have%20up%20to%20six.,-Desktop
![image](https://github.com/user-attachments/assets/29d89bb9-72d9-4021-a915-fe8ce765ca15)

We verified this at our end by adding 9 button actions.

In web chat: All buttons are visible.
![image](https://github.com/user-attachments/assets/06ca59ce-d9fe-47fe-9f91-405f20c6744f)

In Teams client: Only 6 buttons are visible and remaining are visible and remaining actions are shown as secondary actions.

![image](https://github.com/user-attachments/assets/9590a669-84c3-40d8-b3e2-b33f5b2a7d51)

**Comment by deeptishashidharaiah** [comment]
Thanks for quick response @Meghana-MSFT , 
In our case, we dont see that secondary actions to expand remaining buttons
![image](https://github.com/user-attachments/assets/488c5dd1-fbd2-4547-b54b-b55b599f6413)

**Comment by deeptishashidharaiah** [comment]
yes thats the very reason we opened this issue, this issue only occurs in teams app not in web page

**Comment by deeptishashidharaiah** [comment]
@Meghana-MSFT  any update?

**Comment by Meghana-MSFT** [comment]
@deeptishashidharaiah - Could you please let us know if you are checking in Teams desktop or Teams web client?

**Comment by deeptishashidharaiah** [comment]
My experience is same for teams web(https://teams.microsoft.com/v2/)  or teams app

**Comment by Meghana-MSFT** [comment]
@deeptishashidharaiah - We have raised a bug for this, we will keep you posted on the updates. Thank you.

**Comment by deeptishashidharaiah** [comment]
Thank you! lets keep this ticket open till then

**Comment by Meghana-MSFT** [comment]
@deeptishashidharaiah - This bug has been fixed. you should see an ellipsis for the other action buttons in your card. We tested this issue in Teams web (client version is 1415/24110115716.) and Desktop (Microsoft Teams version 24341.1300.3292.5164.) clients. The card is rendering properly in both clients.
![image](https://github.com/user-attachments/assets/3adadeaf-c003-47ec-b247-422d6b925937)
